nss-day-cohort-45/holiday-road-cold-shoulders
# dialogGenerator.js is functional

## Changes:

Got the listeners for the details buttons up and working.
Modal is now dynamically generated with basic info.

## To Test:

Select an attraction or eatery then click the details button to make sure the modal is showing properly.
